src/shogun/labels/MappedBinaryLabels.cpp

```diff
@@ -0,0 +1,50 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Authors: Heiko Strathmann
+ */
+
+#include <shogun/labels/MappedBinaryLabels.h>
+#include <shogun/labels/MulticlassLabels.h>
+
+
+using namespace shogun;
+
+std::tuple<LabelMap, LabelMap> CMappedBinaryLabels::create_mapping(const CLabels* orig) const
```

Comment:
std::pair? not sure if it makes any performance difference, but then you can use .first and .second instead of std::get<>()

Reply:
ah yes thx, I had another thing in there previously.